brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaNodeToEntityConverter.java

```diff
@@ -95,6 +105,8 @@ public ToscaNodeToEntityConverter setNodeId(String nodeId) {
         // Add TOSCA node type as a property
         spec.configure("tosca.node.type", this.nodeTemplate.getType());
         spec.configure("tosca.template.id", this.nodeId);
+        // Use the nodeId as the camp.template.id to enable DSL lookup
+        spec.configure(BrooklynCampConstants.PLAN_ID, this.nodeId);
```

Comment:
Probably, we should to decide a generic ID for represent plan ids, avoiding "TOSCA" or "CAMP" string. Or adding "tosca.template.id" [here](https://github.com/apache/incubator-brooklyn/blob/master/usage/camp/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/dsl/methods/DslComponent.java#L127)